apps/mobile/src/screens/home/hooks/use-visit-data.ts

```diff
@@ -0,0 +1,67 @@
+import { useAuth } from '@clerk/clerk-expo'
+import { createRoute } from 'agrume'
+import * as DI from 'diabolo'
+import { useEffect, useState } from 'react'
+
+import { visitHistoryService } from '../../../services/visit-history-service/visit-history-service'
+import { serverImpls } from '../../../shared/utils/server-impls'
+import type { VisitWithJourneyAndMuseum } from '../../../../../../packages/db/src/db'
```

Comment:
Why you import with relative path? The goal of the `packages` is folder is to avoid this for shared packages. You should just use `db`.